code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -679,6 +679,9 @@
 		return
 
 	var/mob/living/carbon/human/T = G.affecting
+	if(isipc(T))
+		to_chat(src, "<span class='warning'>[T] is not a creature you can enthrall.</span>")
+
```

Comment:
You need to add `return` here otherwise it will just show message, but still enthrall